jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -475,6 +479,10 @@ def _on_viewer_change(self, msg={}):
         if hasattr(self, 'viewer') and ref_data is not None:
             if ref_data.label in self.orientation.choices:
                 self.orientation.selected = ref_data.label
+            else:
+                self.orientation.selected = "Default orientation"

```

Comment:
Instead of hardcoding this string, use this var: https://github.com/spacetelescope/jdaviz/blob/deda388472169d11604db304a369869e2e13d00e/jdaviz/configs/imviz/helper.py#L22

Reply:
Done, thanks!